wps-common/src/main/java/au/org/aodn/aws/wps/status/StatusHelper.java

```diff
@@ -1,20 +1,179 @@
 package au.org.aodn.aws.wps.status;
 
+import net.opengis.ows._1.*;
+
+import net.opengis.wps._1_0.*;
+import net.opengis.ows._1.ObjectFactory;
```

Comment:
Is this the wrong import?  Doesn't seem to be used below and references to net.opengis.wps._1_0.ObjectFactory are fully qualified.

Reply:
removed.